src/ui.cpp

```diff
@@ -1456,8 +1456,11 @@ void UI::Run()
     // We need to pass correct HINSTANCE to wxEntry() and the right value is
     // HINSTANCE of this DLL, not of the main .exe.
 
-    if ( !ms_hInstance )
-        return; // DllMain() not called? -- FIXME: throw
+    if ( !ms_hInstance ) {
```

Comment:
Can you please fix this to respect upstream code formatting, i.e. `{` on its own line (see above your changes)? TIA!